app.js

```diff
@@ -1,5 +1,38 @@
 // import functions and grab DOM elements
+import { countsAsYes } from '../utils.js';
+import { percentage } from '../utils.js';
```

Comment:
These can be imported together (separated by a comma) since they are from the same file. :)